python/ray/tune/trainable/util.py

```diff
@@ -339,6 +339,11 @@ def setup(self, config):
                     setup_kwargs[k] = parameter_registry.get(prefix + k)
                 super(_Inner, self).setup(config, **setup_kwargs)
 
+            # Workaround for actor name not being logged correctly
+            # if __repr__ is not directly defined in a class.
+            def __repr__(self):
```

Comment:
ah is this always an issue then?

Reply:
It still hasn't been fixed in core afaik